Code/ForceField/MMFF/Nonbonded.h

```diff
@@ -93,6 +93,7 @@ RDKIT_FORCEFIELD_EXPORT double calcVdWEnergy(const double dist,
                                              const double R_star_ij,
                                              const double wellDepth);
 //! calculates and returns the electrostatic MMFF energy
+// FIX: idx1 and idx2 are not used
 RDKIT_FORCEFIELD_EXPORT double calcEleEnergy(unsigned int idx1,
```

Comment:
I did see a suggestion that for header files commenting out names is a good compromise

```
RDKIT_FORCEFIELD_EXPORT double calcEleEnergy(unsigned int /* idx1 */,
                                             unsigned int /* idx2 */)
```

Reply:
I definitely like leaving them in declarations in the header file so that we can guess what the intent is.
I'm not thrilled with the way the commented out form looks, so as long as having them uncommented in the declarations doesn't generate warnings, I'd rather not mess with that side.